backend/ibutsu_server/controllers/import_controller.py

```diff
@@ -89,4 +90,4 @@ def add_import(
         run_archive_import.delay(new_import.to_dict())
     else:
         return "Unsupported Media Type", 415
-    return new_import.to_dict(), 202
+    return new_import.to_dict(), 2
```

Comment:
```suggestion
    return new_import.to_dict(), HTTPStatus.ACCEPTED
```
If we change this line shouldn't it be with HTTPStatus?

Reply:
great eye and catch! This is definitely a typo or bad find+replace entry.